src/Android/Renderers/CustomTabbedRenderer.cs

```diff
@@ -49,7 +49,7 @@ private BottomNavigationView GetBottomNavigationView()
             return null;
         }
 
-        async void BottomNavigationView.IOnNavigationItemReselectedListener.OnNavigationItemReselected(IMenuItem item)
+        public async void OnNavigationItemReselected(IMenuItem item)
```

Comment:
Should this be `async void` or `async Task`? _Feels_ like we need `async Task` to avoid latent exceptions or app crashes that aren't handled through this async call.

Reply:
Looks like we don't have a choice...

![image](https://user-images.githubusercontent.com/59324545/134517553-0ae8909b-d106-49af-b403-112936564d6c.png)

Reply:
Bummer, I didn't realize it was an interface method implementation. With `async void` you'll need to ensure that there's proper error handling/try-catch blocks to ensure exceptions are bubbled up outside of latent execution where we'd lose the stack-trace, etc.